fetch_data.py

```diff
@@ -357,23 +359,45 @@ def parse_meta_xml_str(meta_xml_str):
     return dim_encodings
 
 
-async def get_dim_encodings(cov_id):
+def get_xml_str_between_tags(meta_xml_str, tag):
+    """Get string encapsulated by a known XML tag.
+
+    Arguments:
+        meta_xml_str (str): string representation of the byte XML response from the WCS DescribeCoverage request
+        tag (str): the xml string that encapsulates the desired information, e.g., 'gmlrgrid:coefficients'
+
+    Returns:
+        str_within_tag (str): string encapsulated by the provided XML tag
+    """
+    tag_open = f"<{tag}>"
+    tag_close = "<"
+    str_after_tag = meta_xml_str.split(tag_open)[1]
+    str_within_tag = str_after_tag.split(tag_close)[0]
+    return str_within_tag
+
+
+async def get_dim_encodings(cov_id, scrape=None):
     """Get the dimension encodings that map integer values to descriptive strings from a
     Rasdaman coverage that stores the encodings in a metadata "encodings" attribute. We
     handle exceptions where the coverage we are requesting encodings from does not exist
-    on the backend to prevent Rasdaman work from blocking API development.
+    on the backend to prevent Rasdaman work from blocking API development. We can use the same request to scrape various other parts of the DescribeCoverage XML response, but this optional.
```

Comment:
Minor nitpick, but this and other comments could use some manual word-wrapping.

Reply:
Thanks!! Yeah I think had a return in there that broke the wrapping. This should be fixed now but please double check (it looked ok in the diff to me and in VS Code).

Reply:
No luck on the word-wrapping, unfortunately. Or we might be using the term word-wrapping in different ways. IMO, we should add more line breaks to try to keep comment lines shorter than ~100 characters.

Reply:
We decided that it's perfectly acceptable to have long docstring with no manual line breaks, and it passes Black formatting. So feel free to disregard this comment!